app/src/main/res/layout/content_main.xml

```diff
@@ -10,86 +10,70 @@
 
     <Button
         android:id="@+id/button4"
-        android:layout_width="wrap_content"
+        android:layout_width="134dp"
         android:layout_height="wrap_content"
-        android:layout_marginStart="28dp"
-        android:layout_marginLeft="28dp"
-        android:layout_marginTop="60dp"
+        android:layout_marginStart="16dp"
+        android:layout_marginLeft="16dp"
+        android:layout_marginEnd="44dp"
+        android:layout_marginRight="44dp"
+        android:layout_marginBottom="40dp"
         android:onClick="viewActivityHistory"
-        android:text="View Activities"
+        android:text="@string/view_activities"
+        app:layout_constraintBottom_toTopOf="@+id/txtActivity"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintHorizontal_bias="0.22"
-        app:layout_constraintStart_toEndOf="@+id/button2"
-        app:layout_constraintTop_toBottomOf="@+id/txtTimeSpent" />
+        app:layout_constraintStart_toEndOf="@+id/button2" />
 
     <EditText
         android:id="@+id/txtTimeSpent"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginTop="44dp"
-        android:layout_marginEnd="96dp"
-        android:layout_marginRight="96dp"
         android:ems="10"
-        android:hint="Time Spent(Hours)"
+        android:hint="@string/time_spent_hours"
```

Comment:
strings.xml is a good idea for internationalization, because you can have a different strings.xml file for each language you support.